src/layered_config_tree/main.py

```diff
@@ -346,6 +346,17 @@ def to_dict(self) -> NestedDict:
                 result[name] = child.to_dict()  # type: ignore[assignment]
         return result
 
+    def get(self, key: str, default_value: NodeValue = None) -> LayeredConfigTree | NodeValue:
+        """Return the LayeredConfigTree or value at the key in the outermost layer of the config tree.
+
+        Parameters
+        ----------
+        key
+            The str we look up in the outermost layer of the config tree.
+        """
+        default = LayeredConfigTree() if default_value is None else default_value
+        return self[key] if key in self._children else default
```

Comment:
I don't understand this. So if you don't pass in a default value (which is likely, right?), then it `default` is None and it just returns an instantiated but empty LayeredConfigTree if it can't find the key?

`lct.get("some-bogus-key")` --> LayeredConfigTree()?

Reply:
Yeah that's how it would work. This allows us to chain get calls without erroring out.